src/NUnitEngine/nunit.engine.core.tests/Services/ExtensionManagerTests.cs

```diff
@@ -12,7 +12,7 @@
 using NUnit.Engine.Internal.FileSystemAccess;
 using System.Diagnostics;
 
-namespace NUnit.Engine.Services.Tests
+namespace NUnit.Engine.Services
```

Comment:
Any reason why dropping the `Tests` suffix in the namespace.

Reply:
There is an inconsistency in the use  of namespaces in the tests. In the beginning, we used a Tests sub-namespace but later we started to use the same namespace as the class under test because it's less wordy and unnecessary, given that our test classes are almost all named `SomethingTests`. So I change them when I come across them. At some point we could do a global change as a completely separate PR.

Reply:
OK, I've seen both practices in my coaching work, but I'll stick with the one you prefer.